src/main/java/com/kryptnostic/rhizome/mappers/keys/ByteArrayKeyMapper.java

```diff
@@ -0,0 +1,22 @@
+package com.kryptnostic.rhizome.mappers.keys;
+
+import com.kryptnostic.rhizome.mappers.SelfRegisteringKeyMapper;
+
+public class ByteArrayKeyMapper implements SelfRegisteringKeyMapper<byte[]> {
+
+    @Override
+    public String fromKey( byte[] key ) {
+        return key == null ? null : new String( key );
```

Comment:
Do we know for sure that everything writing this will be using the same default charset? I don't know that it's necessary but it seems like it may be a good idea to explicitly specify a charset here.